package.json

```diff
@@ -1470,7 +1470,7 @@
                 {
                     "command": "aws.ecs.runCommandInContainer",
                     "group": "0@1",
-                    "when": "view == aws.explorer && viewItem == awsEcsContainerNode"
+                    "when": "view == aws.explorer && viewItem =~ /^(awsEcsContainerNodeExecEnabled|awsEcsContainerNodeExecDisabled)$/"
```

Comment:
they both are prefixed with `awsEcsContainerNodeExec`. could match only on that.